cltk/corpus/latin/wordnet.py

```diff
@@ -0,0 +1,2291 @@
+            objects. Synsets whose LCS is the root node of the taxonomy will
+            have a score of 0 (e.g. N['dog'][0] and N['table'][0]).
+
+        >>> from cltk.corpus.latin.wordnet import WordNetCorpusReader, WordNetICCorpusReader
+        >>> LASLA_IC = WordNetICCorpusReader(fileids=['ic-lasla.dat'])
+        >>> LWN = WordNetCorpusReader()
+        >>> s1 = LWN.synset_from_pos_and_offset('n', '02542418')
+        >>> s2 = LWN.synset_from_pos_and_offset('n', '03457380')
+        >>> s1.res_similarity(s2, LASLA_IC)
+        6.056495670686355
+        """
+
+        ic1, ic2, lcs_ic = self._lcs_ic(other, icreader)
+        return lcs_ic
+
+    def jcn_similarity(self, other, icreader, verbose=False):
+        """
+        Jiang-Conrath Similarity:
+        Return a score denoting how similar two word senses are, based on the
+        Information Content (IC) of the Least Common Subsumer (most specific
+        ancestor node) and that of the two input Synsets. The relationship is
+        given by the equation 1 / (IC(s1) + IC(s2) - 2 * IC(lcs)).
+
+        :type  other: Synset
+        :param other: The ``Synset`` that this ``Synset`` is being compared to.
+        :type  ic: WordNetICCorpusReader
+        :param ic: an information content reader
+        :return: A float score denoting the similarity of the two ``Synset``
+            objects.
+
+        >>> from cltk.corpus.latin.wordnet import WordNetCorpusReader, WordNetICCorpusReader
+        >>> LASLA_IC = WordNetICCorpusReader(fileids=['ic-lasla.dat'])
+        >>> LWN = WordNetCorpusReader()
+        >>> s1 = LWN.synset_from_pos_and_offset('n', '02542418')
+        >>> s2 = LWN.synset_from_pos_and_offset('n', '03457380')
+        >>> s1.jcn_similarity(s2, LASLA_IC)
+        0.23789011550933925
+
+        """
+
+        if self == other:
+            return _INF
+
+        ic1, ic2, lcs_ic = self._lcs_ic(other, icreader)
+
+        # If either of the input synsets are the root synset, or have a
+        # frequency of 0 (sparse data problem), return 0.
+        if ic1 == 0 or ic2 == 0:
+            return 0
+
+        ic_difference = ic1 + ic2 - 2 * lcs_ic
+
+        if ic_difference == 0:
+            return _INF
+
+        return 1 / ic_difference
+
+    def lin_similarity(self, other, icreader, verbose=False):
+        """
+        Lin Similarity:
+        Return a score denoting how similar two word senses are, based on the
+        Information Content (IC) of the Least Common Subsumer (most specific
+        ancestor node) and that of the two input Synsets. The relationship is
+        given by the equation 2 * IC(lcs) / (IC(s1) + IC(s2)).
+
+        :type other: Synset
+        :param other: The ``Synset`` that this ``Synset`` is being compared to.
+        :type ic: WordNetICCorpusReader
+        :param ic: an information content reader
+        :return: A float score denoting the similarity of the two ``Synset``
+            objects, in the range 0 to 1.
+
+        >>
```

Comment:
Oh I see you've already thought of this. Send the PR for this now, it'll need to be in `ic-lasla.dat` before merging this PR